.actor/INPUT_SCHEMA.json

```diff
@@ -0,0 +1,95 @@
+{
+    "title": "Snowflake uploader input",
+    "type": "object",
+    "schemaVersion": 1,
+    "properties": {
+        "synchronizeSchema": {
+            "title": "Database schema columns",
+            "type": "array",
+            "description": "SETTING THIS OPTION WILL DROP YOUR TABLE. In case your destination table has a schema incompatible with the dataset, you can provide a list of column names and their types, then the table will be synchronized according to your definitions by dropping and recreating the table. Key is the column name, value is the column type (as in SQL query, like VARCHAR). This is optional if your table already has the desired schema.",
```

Comment:
People don't read the descriptions much. If there is something serious, currently there are 2 ways to show to users more aggressively:
1. Using top-level "description" field which is then displayed on top of the editor
2. Adding it to the "title", e.g. "title": "Database schema columns (SETTING THIS OPTION WILL DROP YOUR TABLE)"

Reply:
Generally, we should first provide required fields like `datasetId` and optional ones later. Since this is optional for everyone who already has the schema in the database, I would move it way below